Source/Fuse.Controls.Panels/GraphicsView.ux.uno

```diff
@@ -332,10 +333,14 @@ namespace Fuse.Controls
 					}
 
 					Internal.DrawManager.PrepareDraw(_dc);
+
 					_dc.PushViewport(this);
 					_dc.PushScissor( new Recti(0, 0, size.X, size.Y) );
 					_dc.Clear(Color);
 
+					if defined(DRAW_RECTS)
```

Comment:
I think `DRAW_RECTS` is too vague of a name (and in a too global of a namespace) for us to use for this. For other functionality, we've used:
- `FUSELIBS_PROFILING`
- `FUSELIBS_GLDEBUG`
- `FUSELIBS_NO_TOASTS`
- `DISABLE_IMPLICIT_GRAPHICSVIEW`
- `USE_HARFBUZZ`
- `USE_JAVASCRIPTCORE` / `USE_V8` / `USE_DUKTAPE`
- `DEBUG_V8`

Even though we haven't been very consistent here, I would prefer if we used something that at least starts with `FUSELIBS_`, so we don't accidentally trample on defines that user-code uses. But perhaps we'd want to say that this is debug-drawing somehow, as opposed to "we're drawing rectangles!"...

So maybe something like `FUSELIBS_DEBUG_DRAW_RECTS` instead?